src/core/components/Root/index.js

```diff
@@ -9,6 +9,8 @@ import type { I18nType } from 'core/types/i18n';
 import type { ReduxStore } from 'core/types/redux';
 import type { ReactRouterHistoryType } from 'core/types/router';
 
+const config = require('config');
```

Comment:
Can you please use this syntax:

```js
import config from 'config';
```

That's probably what's causing the issue.

Also, I think you can "inject" `config` into the component:

```js
const Root = ({ children, history, i18n, store, cookies = null, _config = config }: Props) => (
```

You'll have to update `Props` with `_config: typeof config`, and use `_config` in L28.

Reply:
It looks like Flow is blowing up now:

![image](https://user-images.githubusercontent.com/13009507/48957676-0d8ee900-ef28-11e8-8c6e-ba79f52fa826.png)


Should I go ahead and add this into `stories/setup/Provider.js`?